server/base/src/main/java/org/apache/accumulo/server/conf/CheckCompactionConfig.java

```diff
@@ -106,7 +106,8 @@ public static void validate(AccumuloConfiguration config, Level level)
 
     Set<String> defaultService = Set.of(DEFAULT_COMPACTION_SERVICE_NAME);
     if (servicesConfig.getPlanners().keySet().equals(defaultService)) {
-      log.warn("Only the default compaction service was created - {}", defaultService);
+      log.atLevel(level).log("Only the default compaction service was created - {}",
```

Comment:
Iirc, the original goal of this log message was to alert users that the compaction service might not have been properly created. However, if a custom service was never defined than this just logs endlessly.

Might be a good follow on to try and validate the servicesConfig planners vs what planner prefixes might be set in the properties?  and only log if the there's a difference?

Reply:
We have implementions for a ConditionalLogger. One deduplicates and the other escalates. If you only want to see the message once every N time units, then use the DeduplicatingLogger. If you want to see the message at level X once every N time units, and at level Y the remaining times, then use the EscalatingLogger.

Reply:
> Might be a good follow on to try and validate the servicesConfig planners vs what planner prefixes might be set in the properties? and only log if there's a difference?

Could this be accomplished by adding unit test for CompactionServicesConfig that verifies it fails/throws an exception under these conditions?